spec/gitpoints/points_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
```

Comment:
I see that in the `spec_helper` you've got `require "gitPoints"`. Are you sure that that `require` is working? You might need to adjust your load path.

Also, camelCase for filenames? We raised you better than that.